Lib/email/_header_value_parser.py

```diff
@@ -1769,6 +1771,9 @@ def get_name_addr(value):
     name_addr = NameAddr()
     # Both the optional display name and the angle-addr can start with cfws.
     leader = None
+    if not value:
+        raise errors.HeaderParseError(
+            "expected name-addr but but found '{}'".format(value))
```

Comment:
It seems that this check is redundant, this function is never called with empty value. But let's leave it just in case.